environment/src/main/java/com/sequenceiq/environment/environment/flow/creation/handler/NetworkCreationHandler.java

```diff
@@ -118,6 +120,12 @@ private void createCloudNetworkIfNeeded(EnvironmentDto environmentDto, Environme
         }
     }
 
+    private void createProviderSpecificNetworkResourcesIfNeeded(EnvironmentDto environmentDto, BaseNetwork network) {
+        if (AZURE.equalsIgnoreCase(environmentDto.getCloudPlatform())) {
```

Comment:
as mentioned above this could be removed if we change the `DefaultNetworkConnector`